media/media.js

```diff
@@ -24,8 +35,8 @@ function updateSingleSelectGroup(selector, className, selectedClassName, state)
 // Validation errors will be hidden
 function resetValidationErrors() {
   document.querySelectorAll('span.errorMessage').forEach((box) => {
-    box.classList.remove('hide');
-    box.classList.add('hide');
+    display(box);
```

Comment:
What is going on here? We display it and then we hide it? Not sure I can follow.